wmcontent.info.yml

```diff
@@ -2,7 +2,7 @@ name: wmcontent
 type: module
 description: Adds configurable entity containers to entity types (for e.g. paragraphs)
 version: 2.2.0
-core_version_requirement: ^10
+core_version_requirement: ">=10"
```

Comment:
Doesn't this mean we also support Drupal 11 _the moment_ it gets released?
Shouldn't we keep `^10` and when Drupal 11 is available and this module has been tested, change it to `^10 || ^11`?

Reply:
@RobinHoutevelts You're right, I might have misinterpreted these warnings 🤔 

```
modules/contrib/wmcontent/wmcontent.info.yml:
┌──────────┬──────┬──────────────────────────────────────────────────────────┐
│  STATUS  │ LINE │                         MESSAGE                          │
├──────────┼──────┼──────────────────────────────────────────────────────────┤
│ Check    │ 0    │ Value of core_version_requirement: ^10 is not compatible │
│ manually │      │ with the next major version of Drupal core. See          │
│          │      │ https://drupal.org/node/3070687.                         │
│          │      │                                                          │
└──────────┴──────┴──────────────────────────────────────────────────────────┘

modules/contrib/wmcontent/composer.json:
┌──────────┬──────┬──────────────────────────────────────────────────────────────┐
│  STATUS  │ LINE │                           MESSAGE                            │
├──────────┼──────┼──────────────────────────────────────────────────────────────┤
│ Check    │ 0    │ The drupal/core requirement is not compatible with the next  │
│ manually │      │ major version of Drupal. Either remove it or update it to be │
│          │      │ compatible. See                                              │
│          │      │ https://drupal.org/node/2514612#s-drupal-9-compatibility.    │
│          │      │                                                              │
└──────────┴──────┴──────────────────────────────────────────────────────────────┘
```

Reply:
Ah yeah, the upgrade_status module is already aiming for Drupal 11.
But I think it's too soon to act on those yet.

Drupal 10.3 is set to be released soonish and will have a lot more deprecations.

Reply:
I've removed Drupal version requirement change 👍